server/test/check.go

```diff
@@ -1251,6 +1012,42 @@ var checkQueryTests = []checkQueryTest{
 			Allowed: true,
 		},
 	},
+	{
```

Comment:
These are the tests for the new functionality that validates the type and relation if the user is a userset.